backend/metadata/databases/default/tables/core_suggestions.yaml

```diff
@@ -0,0 +1,70 @@
+table:
+  schema: core
+  name: suggestions
+object_relationships:
+  - name: section
+    using:
+      manual_configuration:
+        remote_table:
+          schema: core
+          name: sections
+        insertion_order: null
+        column_mapping:
+          section_id: id
+insert_permissions:
+  - role: authenticated
+    permission:
+      check:
+        section:
+          agreement:
+            group:
+              group_users:
+                user_id:
+                  _eq: X-Hasura-User-Id
```

Comment:
This wouldn't work, because a `group_users➡️user_id` is equal to the id column of users table, whereas `X-Hasura-User-Id` is equal to the `user_id` column of the users table.
So we need to dive in a bit deeper, and use `group_users➡️user➡️user_id`. 
Same goes for select and update permissions.

Reply:
Aha, thank you for explaining that! In other words, user.user_id is the external user _account_ ID, which of course is distinct from the internal user.id. Is it too late to rename user.user_id to user_account_id so this distinction is clear in the data model?

Reply:
Exactly.
Right, the confusion is understandable, but I think that it's indeed too late to change the column name.